src/cdn.cls.php

```diff
@@ -311,7 +311,10 @@ private function _replace_inline_css()
 			$url = str_replace(array(' ', '\t', '\n', '\r', '\0', '\x0B', '"', "'", '&quot;', '&#039;'), '', $url);
 
 			// Parse file postfix
-			$postfix = '.' . pathinfo(parse_url($url, PHP_URL_PATH), PATHINFO_EXTENSION);
+			$parsed_url = parse_url($url, PHP_URL_PATH);
+			if(is_null($parsed_url)) continue;
```

Comment:
maybe you can just check `if(!$parse_url)`?

Reply:
Fixed